src/seat.c

```diff
@@ -515,6 +522,9 @@ seat_reconfigure(struct server *server)
 		case WLR_INPUT_DEVICE_POINTER:
 			configure_libinput(input->wlr_input_device);
 			break;
+		case WLR_INPUT_DEVICE_TOUCH:
+			map_touch_to_output(seat, input->wlr_input_device);
```

Comment:
I guess we should also call `configure_libinput(input->wlr_input_device);` here.